### PR TITLE
Frontend for Application steps popup

### PR DESCRIPTION
**Pull request by fzjawed**

### CODE REVIEW

File: launchpad-frontend/package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "launchpad",
   "version": "0.1.0",
   "private": true,
-  "proxy": "http://localhost:5000",
+  "proxy": "http://localhost:5001",
```
Comment (zanq-mw) [inline launchpad-frontend/package.json:5]: whyd u change this lol

File: launchpad_backend/run.py [hunk 2]
```diff
@@ -3,4 +3,4 @@
 
 # start the backend server
 if __name__ == "__main__":
-    app.run(host="localhost", port=5000, debug=True)
+    app.run(host="localhost", port=5001, debug=True)
```
Comment (zanq-mw) [inline launchpad_backend/run.py:6]: if u change it back to 5000, make sure to fix it here too

File: launchpad-frontend/src/jobsPage/jobsPage.js [hunk 3]
```diff
@@ -393,9 +393,9 @@ function JobExpanded({ postingSelected, updateList, postings, saved }) {
                     alignContent: "space-evenly",
                   }}
                 >
-                  <Button variant="contained" sx={PageStyles.apply}>
+                  <ApplyButton companyName={expandedPosting.companyName}>
```
Comment (zanq-mw) [inline launchpad-frontend/src/jobsPage/jobsPage.js:396]: im getting an error cuz of this. I think you have to import ApplyButton into this file

File: launchpad-frontend/package.json (hunk 1 above)
Comment (fzjawed) [reply launchpad-frontend/package.json:5]: nikki told me to ;-; cuz it wasn't running properly she said sth about the ports on mac being different </3 sry

File: launchpad-frontend/src/jobsPage/jobsPage.js (hunk 3 above)
Comment (fzjawed) [reply launchpad-frontend/src/jobsPage/jobsPage.js:397]: done!! thank u for noticing 🥳

File: launchpad-frontend/package.json (hunk 1 above)
Comment (nikkikapadia) [reply launchpad-frontend/package.json:5]: leave this as 5000 for now we can discuss what port to change it to next standup.

File: launchpad_backend/run.py (hunk 2 above)
Comment (nikkikapadia) [reply launchpad_backend/run.py:6]: same with this change it back to 5000 for now :)

Comment (fzjawed) [reply launchpad_backend/run.py:6]: okii

File: launchpad-frontend/package.json (hunk 1 above)
Comment (fzjawed) [reply launchpad-frontend/package.json]: yez ma'am

**Review by zanq-mw — APPROVED** [review]

**Pull request merged**